docs/CompVis_Stable_Diffusion_Instructions.md

```diff
@@ -0,0 +1,77 @@
+# Running CompVis Stable Diffusion on a Single GPU with ONNX Runtime and DirectML
+
+This instruction downloads and set up CompVis Stable Diffusion v1.4 model through the Hugging Face diffusers and transformers library. It pulls relevant Python packages that allows the model to run on most discrete consumer graphics GPUs with ONNX Runtime on the DirectML execution provider. This instruction is based on the prior work of [Neil McAlister](https://www.travelneil.com/stable-diffusion-windows-amd.html) with the more up-to-date script version from the Hugging Face Diffusers repo and its dependency packages as well as additional conversion steps for better execution performance.
+
+## Installing Dependency Packages
+
+We need a few Python packages, namely the Hugging Face script libraries for transformers and diffusers along with ONNX Runtime for DirectML.
+
+```
+pip install diffusers transformers onnxruntime-directml onnx
+```
+
+### Hardware Requirement
+Since it is still necessary for the entire model to fit within the GPU memory while executing. An ideal GPU to run this model with should have 8GB of VRAM available. Here are a few examples:
```

Comment:
Since the entire model must fit within the GPU memory while executing, a GPU should have at least 8GB of VRAM available to run this model. Here are a few examples: